examples/tests/linuxbuild.py

```diff
@@ -24,7 +24,8 @@ def setUp(self):
 
         self.linux_build = kernel.KernelBuild(kernel_version,
                                               linux_config,
-                                              self.srcdir)
+                                              self.srcdir,
+                                              ['/var/tmp'])
```

Comment:
Sorry, I'm failing to understand what paths would be in `self.cache_dirs` then, since it's not part of the test api. Could you clarify what's the value of `self.cache_dirs` in your example?